Source/Krypton Components/TestForm/Form1.cs

```diff
@@ -92,5 +92,12 @@ private void kbtnIntegratedToolbar_Click(object sender, EventArgs e)
 
             integratedToolBar.Show();
         }
+
+        private void kryptonButton3_Click(object sender, EventArgs e)
+        {
+            KryptonThemeBrowserForm themeBrowser = new();
```

Comment:
The `new` style being used by KT, is for `var` first, then the new to specify the type please ?

Reply:
Usage of full qualifiers